frontend/src/context/NetworkContext.tsx

```diff
@@ -0,0 +1,11 @@
+import { createContext } from "react";
+import { NearNetwork } from "next.config";
+
+export interface NetworkContext {
+  currentNetwork: NearNetwork;
+  networks: NearNetwork[];
+}
+
+export const NetworkContext = createContext<NetworkContext | undefined>(
```

Comment:
We don't need an explicit provider, simple context would be fine (and minimalistic)